prow/cmd/mkpod/main.go

```diff
@@ -131,6 +132,17 @@ func main() {
 		}
 	}
 
+	// We need to remove the created-by-prow label, otherwise sinker will promptly clean this
```

Comment:
I guess kubernetes will eventually clean it up automatically and we expect the volume of these pods created within that duration to be insignificant?